README.md

```diff
@@ -17,7 +17,7 @@ Install
 =======
 
 ```
-conda install -c biocore VSEARCH MAFFT SortMeRNA numpy
+conda install -c biocore deblur
```

Comment:
testing this now. btw, i noticed that the deblur package from conda is ~85MB. which is really large. the repo isn't that size, so was a bit surprised. Any idea why its so large?